src/ProxyManager/Generator/Util/IdentifierSuffixer.php

```diff
@@ -48,6 +49,10 @@ public static function getIdentifier(string $name): string
 
     private static function loadBaseHashSalt(): string
     {
-        return sha1(serialize(InstalledVersions::getRawData()));
+        return sha1(serialize(
+            is_callable([InstalledVersions::class, 'getAllRawData'])
+                ? InstalledVersions::getAllRawData() // Composer >= 2.0.14
```

Comment:
Note: I'll add a `"conflict": {"composer/composer": "<2.0.14"}` before merging, and remove the conditional.